test/controllers/hosts_controller_test.rb

```diff
@@ -811,7 +811,7 @@ def test_unset_manage
       :host_ids => Host.pluck('hosts.id'),
     }, session: set_session_user
     assert_redirected_to :controller => :hosts, :action => :index
-    assert flash[:error] == "Cannot update Location to Location 1 because of mismatch in settings"
+    assert flash[:error] == "Cannot update location to Location 1 because of mismatch in settings"
```

Comment:
Since it's failing, could you update it as well so the failure message is useful?
```suggestion
    assert_equal "Cannot update location to Location 1 because of mismatch in settings", flash[:error]
```

Reply:
I can change the assert style. But I don't see how it changes anything about the failure message.

Reply:
Didn't you see the failure? It said that `false` wasn't `true`. With `assert_equal` it will tell you the exact mismatch: both the expected and actual texts. That makes looking at CI results better.